### PR TITLE
i18n(ko-KR): update `integrations-guide/`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- update `integrations-guide/`
- #9240
- #10162
- #10187

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by jsparkdev** [comment]
Once #10177, #10166 are merged, I expect to be able to merge this PR successfully.

**Review by yanthomasdev — APPROVED** [review]
LGTM! I'll merge the umbrella PR once I approve all of the individual PRs.

**Pull request closed without merging**
